frontend/projects/abtesting/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts

```diff
@@ -79,12 +79,14 @@ export class EnrollmentOverTimeComponent implements OnChanges, OnInit, OnDestroy
       this.populateGraphData(graphInfo);
     });
     // Used to fetch last 7 days graph data
-    this.experimentService.setGraphRange(this.selectedDateFilter, this.experiment.id);
+    this.experimentService.setGraphRange(this.selectedDateFilter, this.experiment.id, -new Date().getTimezoneOffset());
   }
 
   // remove empty series data labels
   formateXAxisLabel(value) {
-    return !isNaN(value) ? '' : value.substring(0, 3);
+    if(this.selectedDateFilter === DATE_RANGE.LAST_SEVEN_DAYS)
```

Comment:
@RidhamShah Can you please format this part of code to add curly braces for more readability?

Reply:
code updated